oshc/.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
This file is generated by Pycharm and you shouldn't add it.

Please remove this file and all the files in the `.idea` directory.